utils/seed.js

```diff
@@ -0,0 +1,93 @@
+const controllers = require('../controllers');
+const services = require('../services');
+const { parser, configs } = require('../utils');
+
+const state = {
+  movies: {
+    page: 0,
+    results: 0,
+    totalPages: 0,
+    totalResults: 0,
+    updatedAt: '',
+    hasMore: true,
+  },
+};
+
+const seed = async () => {
+  console.log('Seeding...');
+
+  await seedGenres();
+
+  while (state.movies.hasMore) {
+    await new Promise(resolve => setTimeout(resolve, 15000));
+
+    await seedMovies();
+  }
+
+  // let timer = await setTimeout(async () => {
+  //   console.log('timing...');
+  //   await seedMovies;
+  //   if (state.movies.hasMore) timer = await setTimeout(seedMovies, 10000);
+  //   else clearTimeout(timer);
+  // }, 1000);
+
+  // await seedMovies();
+};
+
+const seedGenres = async () => {
+  const allGenres = await controllers.genre.readAll();
+  if (!allGenres || allGenres.length === 0) {
+    await console.log('Seeding genres...');
+
+    const seed = await services.getGenres();
+
+    for await (let genre of seed.genres) {
+      await controllers.genre.create(await parser.genres(genre));
```

Comment:
No initial `await`, becasue that's the whole point of the `for await`
```suggestion
      controllers.genre.create(await parser.genres(genre));
```